### DISCUSSION TITLE
Close Dialog During Navigation Event

### DISCUSSION THREAD

**Discussion by nernersesyan** [primefaces/community#575] [PrimeNG]
Hello.
In my opinion, it would be very useful if you add an option of closing the dialog during navigation change on DinamicDialogConfig like Angular Material has ('MatDialogConfig', closeOnNavigation: boolean).
Thanks in advance.
![Screenshot 2023-11-29 at 14 30 17](https://github.com/primefaces/community/assets/118472287/4420b361-4abc-495c-b019-6bbd152e017b)
)

**Comment by Hambardzum93** [comment]
This would be a very useful feature, I agree

**Comment by samvel077** [comment]
I also will be happy to see this feature.

**Comment by K-Makaryan** [comment]
I Agree. Every time checking for route changes is not convenient.

**Comment by cetincakiroglu** [comment]
Hi,

Dialog is already closing on navigation since the component is destroyed. Could you please a bit explain what do you want or share a stackblitz example?
